app/src/main/java/de/westnordost/streetcomplete/quests/toilet_availability/AddToiletAvailability.java

```diff
@@ -0,0 +1,36 @@
+package de.westnordost.streetcomplete.quests.toilet_availability;
+
+import android.os.Bundle;
+
+import javax.inject.Inject;
+
+import de.westnordost.streetcomplete.R;
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment;
+
+public class AddToiletAvailability extends SimpleOverpassQuestType
+{
+	@Inject public AddToiletAvailability(OverpassMapDataDao overpassServer) { super(overpassServer); }
+
+	@Override protected String getTagFilters()
+	{
+		return "nodes, ways with (shop ~ mall|department_store and name) and !toilet";
```

Comment:
(It wasn't a problem, that is why I approved the PR already. Superfluous parenthesis are automatically removed)